pkg/provisioningv2/rke2/planner/etcdrestore.go

```diff
@@ -151,9 +151,9 @@ func generateCreateEtcdTombstoneInstruction(controlPlane *rkev1.RKEControlPlane)
 	}
 }
 
-// runEtcdRestoreControlPlaneEtcdServiceStop generates service stop plans for every etcd and controlplane node in the cluster and
+// runEtcdRestoreClusterAgentStopPlan generates service stop plans for every etcd/controlplane and all nodes running a cluster agent in the cluster and
 // assigns/checks the plans to ensure they were successful
-func (p *Planner) runEtcdRestoreControlPlaneEtcdServiceStop(controlPlane *rkev1.RKEControlPlane, snapshot *rkev1.ETCDSnapshot, tokensSecret plan.Secret, clusterPlan *plan.Plan) error {
+func (p *Planner) runEtcdRestoreClusterAgentStopPlan(controlPlane *rkev1.RKEControlPlane, snapshot *rkev1.ETCDSnapshot, tokensSecret plan.Secret, clusterPlan *plan.Plan) error {
```

Comment:
nit: the name of this function should probably be something like `runEtcdRestoreServiceStop`

the name of the plan right now implies it is going to stop the cluster agent, at least to me. While it does this, it also does a bunch of other things.